boot/cmdline.go

```diff
@@ -41,10 +41,13 @@ const (
 	// ModeRecover is a mode in which the device boots into the recovery
 	// system.
 	ModeRecover = "recover"
+	// ModeCloudImg is a mode in which classic cloud image is
+	// booted.
+	ModeCloudImg = "run-cloudimg-rootfs-draft-1"
```

Comment:
what's the plan for this PR, we can't really land these changes as they here in snapd master

Reply:
the plan is that this PR will be built into a forked core-initrd, to be used in forked cloud images for now. It is intentionally worded as not mergable into snapd master at the moment.

Reply:
thank for clarifying this

Reply:
PR has been updated, and imho it is ready for review and merging now.